user_guide_src/source/installation/upgrade_404.rst

```diff
@@ -0,0 +1,19 @@
+#############################
+Upgrading from 4.0.x to 4.0.4
+#############################
+
+CodeIgniter 4.0.4 fixes a bug in the implementation of Controller Filter, breaking
+code implementing the ``FilterInterface``.
+
+**Update FilterInterface declarations**
+
+The method signatures for ``after()`` and ``before()`` need to be update to include ``$arguments``. The function
```

Comment:
Please change `to be update` to `to be updated`.

Reply:
Thanks!